VoxelSniperCore/src/main/java/com/github/kevindagame/VoxelBrushManager.java

```diff
@@ -17,7 +14,7 @@ public class VoxelBrushManager {
 
     private static VoxelBrushManager instance = null;
 
-    private final Multimap<Class<? extends IBrush>, String> brushes = HashMultimap.create();
+    private final Map<String, Class<? extends IBrush>> brushes = new HashMap<>();
     private final List<String> brushHandles = new ArrayList<>();
```

Comment:
Do we still need this List? I feel like we could just use brushes.keySet() instead.

Reply:
this is a good idea. I'll to that.